### PR TITLE
Mod/10886 new table section

### PR DESCRIPTION
**Pull request by brianpetway**
**High level description:**

Add the Award Table section to the new search results page

**Technical details:**

Removed the temp component and added the results table, now using dtui Table

**JIRA Ticket:**
[DEV-10886](https://federal-spending-transparency.atlassian.net/browse/DEV-10886)

**Mockup:**
https://invis.io/RYA3XN5WP#/273832670_Homepage_2-2_E

The following are ALL required for the PR to be merged:

Author:
- [x ] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [ ] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [ ] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/_scss/elements/_table.scss [hunk 1]
```diff
@@ -3,12 +3,24 @@ table {
...
+
+        //&.award-results-table-dtui {
+        //    tbody.usda-table__body {
+        //        .usda-table__row {
+        //            border-left: 0;
+        //            border-right: 0;
+        //            border-bottom: solid rem(1) $gray-cool-10;
+        //        }
+        //    }
+        //}
```
Comment (anjenkin) [inline src/_scss/elements/_table.scss:23]: can this be removed?

File: src/_scss/pages/search/results/searchResults.scss [hunk 2]
```diff
@@ -12,7 +12,7 @@
         @import "../../../elements/_divider";
         @import "./table/resultsTable";
         @import "./screens/screens";
-        @import './tables';
+        //@import './tables';
```
Comment (anjenkin) [inline src/_scss/pages/search/results/searchResults.scss:15]: remove?

**Review by anjenkin — CHANGES_REQUESTED** [review]
table looks good.  see comments inline.  qat needs to be merged into this branch so it's clear what the pr changed from the latest

File: src/_scss/elements/_table.scss (hunk 1 above)
Comment (brianpetway) [reply src/_scss/elements/_table.scss:23]: removed

File: src/_scss/pages/search/results/searchResults.scss (hunk 2 above)
Comment (brianpetway) [reply src/_scss/pages/search/results/searchResults.scss:15]: removed

**Comment by brianpetway** [comment]
Latest qat is merged in, no more merge conflicts, easier to see changes now

**Review by anjenkin — APPROVED** [review]

**Pull request merged**